index.js

```diff
@@ -140,9 +140,9 @@ const parseMMD = /*#__PURE__*/function () {
   };
 }();
 
-module.exports = function mermaidParse(definition, config) {
+module.exports = function mermaidParse(definition, config, puppeteerConfig = null) {
```

Comment:
Can you please add a `@param` in the `index.d.ts` with  a description and type of `puppeteerConfig`?

Also you can initialize the config with `undefined` instead to infer that there it hasn't been defined by default
```suggestion
module.exports = function mermaidParse(definition, config, puppeteerConfig = undefined) {
```